vertical-pod-autoscaler/pkg/recommender/model/cluster_test.go

```diff
@@ -386,3 +388,78 @@ func TestEmptySelector(t *testing.T) {
 	assert.Contains(t, vpa.aggregateContainerStates, cluster.aggregateStateKeyForContainerID(containerID1))
 	assert.Contains(t, vpa.aggregateContainerStates, cluster.aggregateStateKeyForContainerID(containerID2))
 }
+
+func TestRecordRecommendation(t *testing.T) {
+	testTimestamp := time.Unix(1555400, 0)
```

Comment:
Any reason why this particular timestamp?

Reply:
Nah, removed. I didn't notice there is already a testTimestamp available